.env

```diff
@@ -0,0 +1 @@
+MONGO_URI=mongodb://localhost:27017/mentor-match
```

Comment:
Don't commit this file. You should use `.env.local` for your local env.

Reply:
removed from repo